src/gui/torrentoptionsdialog.cpp

```diff
@@ -109,27 +137,61 @@ TorrentOptionsDialog::TorrentOptionsDialog(QWidget *parent, const QVector<BitTor
         if (allSameDHT)
         {
             if (torrent->isDHTDisabled() != isFirstTorrentDHTDisabled)
-            {
-                m_ui->checkDisableDHT->setCheckState(Qt::PartiallyChecked);
                 allSameDHT = false;
-            }
         }
         if (allSamePEX)
         {
             if (torrent->isPEXDisabled() != isFirstTorrentPEXDisabled)
-            {
-                m_ui->checkDisablePEX->setCheckState(Qt::PartiallyChecked);
                 allSamePEX = false;
-            }
         }
         if (allSameLSD)
         {
             if (torrent->isLSDDisabled() != isFirstTorrentLSDDisabled)
-            {
-                m_ui->checkDisableLSD->setCheckState(Qt::PartiallyChecked);
                 allSameLSD = false;
-            }
         }
+        if (allSameSequential)
+        {
+            if (torrent->isSequentialDownload() != isFirstTorrentSequentialEnabled)
+                allSameSequential = false;
+        }
+        if (allSameFirstLastPieces)
+        {
+            if (torrent->hasFirstLastPiecePriority() != isFirstTorrentFirstLastPiecesEnabled)
+                allSameFirstLastPieces = false;
+        }
+    }
+
+    if (allSameAutoTMM)
+        m_ui->checkAutoTMM->setChecked(isFirstTorrentAutoTMMEnabled);
+    else
+        m_ui->checkAutoTMM->setCheckState(Qt::PartiallyChecked);
+
+    if (allSameSavePath)
+        m_ui->savePath->setSelectedPath(firstTorrentSavePath);
+
+    const QString dashes = QLatin1String("--");
+    m_currentCategoriesString = dashes + tr("Currently used categories") + dashes;
+    if (!m_allSameCategory)
+    {
+        m_ui->comboCategory->addItem(m_currentCategoriesString);
+        m_ui->comboCategory->clearEditText();
+        m_ui->comboCategory->lineEdit()->setPlaceholderText(m_currentCategoriesString);
+    }
+    else if (!firstTorrentCategory.isEmpty())
+    {
+        m_ui->comboCategory->setCurrentText(firstTorrentCategory);
+        m_ui->comboCategory->addItem(firstTorrentCategory);
+    }
+    m_ui->comboCategory->addItem("");
```

Comment:
This is an empty item, is it correct?
If it is intended, then try `m_ui->comboCategory->addItem(QString());`

Reply:
All comments addressed.
>This is an empty item, is it correct?

Yes, for "no category".